src/loaders/tasks.js

```diff
@@ -29,10 +30,25 @@ export default ({ queue, index }) => {
       };
     }
   );
+  const taskActions = new DataLoader(queries =>
+    Promise.all(
+      queries.map(async ({ taskGroupId, filter }) => {
+        const url = await queue.buildUrl(
+          queue.getLatestArtifact,
+          taskGroupId,
+          'public/actions.json'
+        );
+        const actions = await (await fetch(url)).json();
```

Comment:
This fetch should retry on 5xx.  And, should it deal with 4xx in any particular way, or is just throwing an exception ok?

Unlike tc-client-web, please do not retry on 4xx :)

Reply:
No need to deal with 4xx in a particular way. GraphQL will make sure to throw the error message.